src/exometer_report_opentsdb.erl

```diff
@@ -100,11 +108,21 @@ exometer_report(_Metric, _DataPoint, _Extra, _Value, St) when St#st.socket =:= u
 
 %% Format a opentsdb output. Each entry is one measurement plus key/value attributes.
 %% put <metric> <time> <measurement> <k1>=<v1> <k2>=<v2>\n
-exometer_report(Metric, DataPoint, _Extra, Value, #st{socket = Sock, hostname = Hostname} = St) ->
-    Line = [
-        "put ", name(Metric), " ", timestamp(), " ", value(Value), " ",
-        "hostname=", Hostname, " ", "type=", metric_elem_to_list(DataPoint), $\n
-    ],
+exometer_report(Metric, DataPoint, _Extra, Value, #st{socket = Sock, hostname = Hostname, join_metric_and_datapoint = JoinMetricAndDatapoint} = St) ->
```

Comment:
I would prefer it if you break the line before "join_metric_and_datapoint =", since it otherwise exceeds the width even of the github "view" window, as well as this diff display. It seems as if adapting to the diff display would impose a width limit of 120.